### ISSUE TITLE
manual_cimoperations.py fails with import issues

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#2039]
tests/manualtest/run_cim_operations.py -h
Traceback (most recent call last):
  File "tests/manualtest/run_cim_operations.py", line 33, in <module>
    from tests.unittest.utils.unittest_extensions import RegexpMixin
  File "/home/kschopmeyer/pywbem/pywbemrequests/pywbem/tests/__init__.py", line 24, in <module>
    from .utils import import_installed
  File "/home/kschopmeyer/pywbem/pywbemrequests/pywbem/tests/utils.py", line 9, in <module>
    import pytest
ModuleNotFoundError: No module named 'pytest'

**Comment by andy-maier** [comment]
Did you run `make develop` in that Python environment?
If so, can you please post the result of `pip list` of the Python environment you saw this behavior?

**Comment by KSchopmeyer** [comment]
Close as no fix required.

**Comment by KSchopmeyer** [comment]
Sorry for my earlier conclusion.  Retried tests with make and same result:

makefile: Done creating the distribution archive files: dist/pywbem-1.0.0.dev1-py2.py3-none-any.whl dist/pywbem-1.0.0.dev1.tar.gz
makefile: Target build done.
(pywbemrequests3) kschopmeyer@leonard:~/pywbem/pywbemrequests/pywbem((HEAD detached at origin/andy/keybinding-type))$ tests/manualtest/run_cim_operations.py http://localhost ks%ks
Testing pywbem version 1.0.0.dev1 from /home/kschopmeyer/pywbem/pywbemrequests/pywbem/pywbem
Traceback (most recent call last):
  File "tests/manualtest/run_cim_operations.py", line 36, in <module>
    from ..utils import import_installed
ValueError: attempted relative import beyond top-level package
(pywbemrequests3) kschopmeyer@leonard:~/pywbem/pywbemrequests/pywbem((HEAD detached at origin/andy/keybinding-type))$

**Comment by andy-maier** [comment]
I guess I screwed that one up in PR #1806. where I assumed the approach for the other testcases would simply work there as well. Which it does not, because these tests are run as independent scripts, whereas the other tests are loaded by py.test.

The fix is to:

1. change relative imports of modules in the tests subtree to become absolute imports starting with the "tests" package,
2. To invoke these scripts with the current directory in the Python package path, e.g.:
    ```
    $ PYTHONPATH=. tests/manualtests/run_cim_operations.py ...
    ```

I can make change 1, and see if we have a good place to document change 2.

**Comment by andy-maier** [comment]
PR #2083 fixes this in master and PR #2084 in 0.17.0.
